content/en/docs/kubernetes/getting-started.md

```diff
@@ -0,0 +1,277 @@
+enhance it in meaningful ways.
+
+To collect all the data, we'll need two installations of the collector, one as a
+[Daemonset](../../collector/deployment/agent) and one as a
+[Deployment](../../collector/deployment/gateway). The Daemonset installation of
+the collector will be used to collect telemetry emitted by services, logs, and
+metrics for nodes, pods, and containers. The deployment installation of the
+collector will be used to collect metrics for the cluster and events.
+
+To install the collector, we'll use the
+[OpenTelemetry Collector Helm chart](../helm/collector), which comes with a few
+configuration options that will make configure the collector easier. If you're
+unfamiliar with Helm, check out [the Helm project site](https://helm.sh/). If
+you're interested in using a Kubernetes operator, see
+[OpenTelemetry Operator](../operator/), but this guide will focus on the Helm
+chart.
+
+## Preparation
+
+This guide will assume the use of a [Kind cluster](https://kind.sigs.k8s.io/),
+but you're free to use any Kubernetes cluster that you deem fit.
+
+Assuming you already have
+[Kind installed](https://kind.sigs.k8s.io/#installation-and-usage), create a new
+kind cluster:
+
+```sh
+kind create cluster
+```
+
+Assuming you already have [Helm installed](https://helm.sh/docs/intro/install/),
+add the OpenTelemetry Collector Helm chart so it can be installed later.
+
+```sh
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+```
+
+## Daemonset Collector
+
+The first step to collecting Kubernetes telemetry is to deploy a daemonset
+instance of the OpenTelemetry Collector to gather telemetry related to nodes and
+workloads running on those node. A daemonset is used to guarantee that this
+instance of the collector is installed on all nodes. Each instance of the
+collector in the daemonset will collect data only from the node on which it is
+running.
+
+This instance of the collector will use the following components:
+
+- [OTLP Receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver):
+  to collect application traces, metrics and logs.
+- [Kubernetes Attributes Processor](../collector/components#kubernetes-attributes-processor):
+  to add Kubernetes metadata to incoming application telemetry.
+- [Kubeletstats Receiver](../collector/components#kubeletstats-receiver): to
+  pull node, pod, and container metrics from the API server on a kubelet.
+- [Filelog Receiver](../collector/components#filelog-receiver): to collect
+  Kubernetes logs and application logs written to stdout/stderr.
+
+Lets break these down.
+
+**OTLP Receiver**
+
+The
+[OTLP Receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver)
+is the best solution for collecting traces, metrics, and logs in the
+[OTLP format](https://opentelemetry.io/docs/specs/otel/protocol/). If you are
```

Comment:
```suggestion
[OTLP format](/docs/specs/otel/protocol/). If you are
```

Reply:
I'm going to accept the suggestion given above because it fixes a build warning, and then push a commit with the correct Docsy submodule reference.